### PR TITLE
Prepare a 0.8.0 release.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,13 @@
 * The `lrlex::build_lex` function has been deprecated in favour of
   `LRNonStreamingLexerDef::from_str`.
 
+## Bug fixes
+
+* The statetable and other elements that were previously included in the user
```
Comment (ptersilie) [inline CHANGES.md:21]: remove "that" before "were".

Comment (ltratt) [reply CHANGES.md]: Fixed in https://github.com/softdevteam/grmtools/pull/189/commits/85117e7c1b47ff3bf2c1cd1b1028f4360fbd0ddd.

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**